assets/css/admin.scss

```diff
@@ -65,7 +66,7 @@
 	}
 
 	.addons-featured {
-		max-width: 1140px;
+		margin: 0 1em 0 0;
```

Comment:
this margin is off from the rest of the page: 
<img width="152" alt="screen shot 2017-12-08 at 8 56 45 am" src="https://user-images.githubusercontent.com/6817400/33768890-e9e3c3fe-dbf5-11e7-9842-07674600ce7a.png">

I may need to fix the search bar.